composer.json

```diff
@@ -86,11 +86,11 @@
         "drupal/field_group": "^3.0",
         "drupal/layout_builder_restrictions": "^2.7",
         "drupal/menu_block": "^1.6",
-        "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
+        "drupal/page_manager": "^4.0",
         "drupal/paragraphs": "^1.11",
         "drupal/pathauto": "^1.8",
         "drupal/rabbit_hole": "^1.0-beta6",
-        "drupal/taxonomy_menu": "dev-3.x#c0892fa9b428ceb784876554c9b80f49d72a246f",
+        "drupal/taxonomy_menu": "dev-3.x",
```

Comment:
If we're on the dev branch I'd rather see the hash with it as well.

Reply:
What if we did that in the profile instead of the module? easier dependency management

Reply:
We should be good citizens and keep it locked here as we do share these modules across multiple projects and multiple profiles.